docs/2023/DDP/Set up a monitoring system/updates/2023-07-21.md

```diff
@@ -4,13 +4,13 @@ author: Jayanth Kumar
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [ ] Install and configure Grafana for infrastructure metrics monitoring on EC2.
```

Comment:
add `x` inside the `[ ]` for the milestones you have achieved

Reply:
Ack